src/main/java/slimeknights/tconstruct/tables/inventory/table/PartBuilderContainer.java

```diff
@@ -65,80 +85,67 @@ public PartBuilderContainer(int id, PlayerInventory inv, PacketBuffer buf) {
 
   @Override
   public void onCraftMatrixChanged(IInventory inventoryIn) {
+    ItemStack itemstack = this.patternSlot.getStack();
```

Comment:
Store material recipe into a variable, then let the interface grab it in the recipe.